daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/EngineConfig.scala

```diff
@@ -17,6 +17,7 @@ import com.daml.lf.transaction.ContractKeyUniquenessMode
   * @param allowedLanguageVersions The range of language versions the
   *     engine is allowed to load.  The engine will crash if it asked
   *     to load a language version that is not included in this range.
+  * @param transactionNormalization Normalizes values according to the LF version
```

Comment:
```suggestion
  * @param transactionNormalization Normalizes transaction nodes according to the LF version
```
byKey is not part of the value

Reply:
done